pkg/dotaws/credentials.go

```diff
@@ -1,55 +0,0 @@
-package dotaws
-
-import (
-	"os"
-
-	"github.com/go-ini/ini"
```

Comment:
I think reference to this package in `Gopkg.toml` (and lock) can be removed now.

Reply:
So it seems like AWS still relies on go-ini. I removed the constraint on it since we do not need a specific version and will just let `dep` do its magic

Reply:
Oh, I see... Thanks for investigation!